src/jetstream_types.rs

```diff
@@ -135,6 +150,15 @@ pub struct ConsumerConfig {
     /// this consumer.
     #[serde(default, skip_serializing_if = "is_default")]
     pub max_ack_pending: i64,
+    /// Only deliver headers without payloads.
+    #[serde(default, skip_serializing_if = "is_default")]
+    pub headers_only: bool,
+    /// Enable flow control messages
+    #[serde(default, skip_serializing_if = "is_default")]
+    pub flow_control: bool,
+    /// Enable idle heartbeat messages
+    #[serde(default, skip_serializing_if = "is_default")]
+    pub idle_heartbeat: i64,
```

Comment:
This is signed altho it looks odd, because time.Duration in Go is signed as far as I know.

Reply:
yep, Go's `time.Duration` is int64, signed.

Reply:
Should we use chrono's Duration for ease of use for our users?

Reply:
Or normal time Duration even.

Reply:
Wrote and added serde_nanos to handle serialisation, works for both chrono and std durations but went with std.